lms/static/my.js

```diff
@@ -105,14 +105,48 @@ function trackReadAllNotificationsButton(button) {
   });
 }
 
+function getPostUploadMessage() {
+  const myDropzone = Dropzone.forElement('#demo-upload');
+  const feedbacks = document.getElementById('upload-feedbacks');
+  const matchesSpan = document.getElementById('upload-matches');
+  const missesSpan = document.getElementById('upload-misses');
+  myDropzone.on('success', function() {
+    const uploadStatus = Array.from(arguments).slice(1)[0];
+    const matches = uploadStatus['exercise_matches'];
+    const misses = uploadStatus['exercise_misses'];
+    if (!feedbacks.classList.contains('feedback-hidden')) {
+      feedbacks.classList.add('feedback-hidden');
+    }
+    if (matches.length) {
+      matchesSpan.innerText += matches + ',';
+    }
+    if (misses.length) {
+      missesSpan.innerText += misses + ',';
+    }
+    feedbacks.classList.add('feedback-transition');
+    feedbacks.clientWidth;
```

Comment:
What is the purpose of this line?

Reply:
Unless we do it - there won't be a transition

Reply:
Wow. That's kind of a strange behavior.
Can you please document it?

Reply:
Added a comment in the end of the line